src/v/raft/consensus.cc

```diff
@@ -170,6 +170,18 @@ void consensus::setup_metrics() {
                          "joint state i.e. configuration is being changed"),
          labels)
          .aggregate(aggregate_labels)});
+
+    if (_recovery_throttle.has_value()) {
+    _metrics.add_group(
+      prometheus_sanitize::metrics_name("raft:recovery"),
+      {sm::make_gauge(
+         "recovery_utilization_available_units",
+         [this] { return _recovery_throttle->get().get_current_units(); },
+         sm::description("Number of available units in recovery"),
```

Comment:
Metric name and description are a bit too low-level to be user-friendly :) How about `learner_recovery_available_bandwidth` (though that's low-level too, maybe `partition_movement_available_bandwidth`?).

Also we should mention the units (bytes/sec) in the description.

Reply:
fixed